src/App.js

```diff
@@ -1,26 +1,119 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Header from './Components/Header';
+import List from './Components/List';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+             todoList: [],
+
+        };
+
+
+
+        this.deleteItem = (deleteItem)=> {
+                this.setState(prev => {
+
+                    return{todoList: prev.todoList.filter((item) => {
+                            return item.key !== deleteItem.key;
+                        }
+                    )}
+                })
+            };
+
+            this.addToList = (itemName) => {
+                const item = {
+                    name: itemName,
+                    chacked: false,
+                    key: Date.now()
```

Comment:
```suggestion
                    id: Date.now()
```
It's more predictable to use id here